common/snapshots.py

```diff
@@ -2161,17 +2174,13 @@ def fileInfo(self):
 
     @fileInfo.setter
     def fileInfo(self, d):
-        assert isinstance(d, dict), 'd is not dict type: {}'.format(d)
+        assert isinstance(d, FileInfoDict), 'd is not FileInfoDict type: {}'.format(d)
         with bz2.BZ2File(self.path(self.FILEINFO), 'wb') as f:
             for path, info in d.items():
-                assert isinstance(path, str), 'path is not str type: {}'.format(path)
```

Comment:
I removed these asserts because we're already asserting that ```d``` is a ```FileInfoDict```, and part of the contract of using its API is that its key is ```bytes``` and values are ```(int, bytes, bytes)```.